src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/service/VisitService.kt

```diff
@@ -139,13 +137,11 @@ class VisitService(
     visitEntity ?: throw VisitNotFoundException("Visit reference $reference not found")
 
     visitEntity.visitStatus = VisitStatus.CANCELLED
-
-    val visitOutCome = createVisitNote(visitEntity, VISIT_OUTCOMES, cancelOutcome.outcome.name)
-    visitEntity.visitNotes.add(visitOutCome)
+    visitEntity.outcomeStatus = cancelOutcome.outcomeStatus
 
     cancelOutcome.text?.let {
-      val statusChangeReason = createVisitNote(visitEntity, STATUS_CHANGED_REASON, cancelOutcome.text)
-      visitEntity.visitNotes.add(statusChangeReason)
+      val outcomeNote = createVisitNote(visitEntity, VisitNoteType.VISIT_OUTCOMES, cancelOutcome.text)
```

Comment:
Mappings table will need updating to reflect the name change

Reply:
are you talking about a document or table?

Reply:
document with the mapping table